_sass/blocks/revenues/_revenues_subpage_steps.scss

```diff
@@ -129,11 +131,17 @@ $onshore-renewables-secondary-color: $blue;
     }
   }
 
-.revenues_subpage-steps_heading {
+.revenues_subpage-steps_heading,
+.revenues_subpage-steps_heading_more {
 	@extend .revenues_steps-heading;
   display: inline-block;
   margin: 0;
 }
+
+.revenues_subpage-steps_heading_more {
```

Comment:
Selector `revenues_subpage-steps_heading_more` should be written in lowercase with hyphens